### DISCUSSION TITLE
Customize theme

### DISCUSSION THREAD

**Discussion by sxryadipta** [DenverCoder1/github-readme-streak-stats#705] [Q&A]
I'm new to programming, so I wish to know how to customize the color of themes? Or how to create your own theme to submit a pull request?

**Comment by DenverCoder1** [comment]
On the website https://streak-stats.demolab.com/demo/ you can open Advanced Options to get a UI for changing the color query parameters. You can add properties and then select a color for each one you add.

![image](https://github.com/user-attachments/assets/35fdc4cb-87f8-48bc-ac82-9cc3b899cf6a)

After filling out your username and other options, you can use your custom colors by copying the url/markdown.

![image](https://github.com/user-attachments/assets/045e6073-e024-4f8e-a865-ba5bfa7ac33e)


If you want to contribute a theme, at the end, you can click Export to PHP which will display the PHP code needed to be added to `themes.php`. More info here - https://github.com/DenverCoder1/github-readme-streak-stats/issues/32

![image](https://github.com/user-attachments/assets/e5d63873-9251-461c-a776-23b7f122d7fe)
